projects/dxc-ngx-cdk/src/lib/dxc-file-input/dxc-file-input.component.ts

```diff
@@ -277,10 +300,127 @@ export class DxcFileInputComponent implements OnChanges, OnInit {
         this.service.emptyArrayFiles();
       }
       this.getPreviewsFiles(event.target.files);
+      this.processFiles(event.target.files);
       event.target.value = "";
     }
   }
 
+    /**
+   * File upload logic to send file as chunk and receive response.
+   * @param event
+   */
+    processFiles(event) {
+    this.files=event;
+    this.fileEventType = EventType.UPLOAD;
+    for(let i=0;i<event.length;i++)
+    {
+      this.uploadChunkDoc(event[i])
+    }
+  }
+
+  UploadFile(eventFiles) {
+  this.uploadChunkDoc(eventFiles);
+}
+uploadChunkDoc(file) {
+  let lastChunksize = 0;
+  this.fileDataUpload = new FileMetaData();
+  this.GUID = uuidv4();
+  this.globalActualChunkCount = 0;
+  this.globalChunkCount = file.size % 1000000 == 0 ? file.size / 1000000 : Math.floor(file.size / 1000000) + 1;
```

Comment:
defined by configuration and input in component

Reply:
this change will be accomodated in next story